### PR TITLE
Make a persistance server

### PR DESCRIPTION
**Pull request by lologarithm**
In reference to: https://github.com/jerold/Catan/issues/11

I started making a websocket server that can accept messages. Next step is to decide how to persist (maybe first pass will just store on disk).

Then create client bindings to call out to the server.

### CODE REVIEW

**Comment by lologarithm** [comment]
ITS READY @jerold

**Comment by lologarithm** [comment]
Ok, did a little cleanup to make things nicer.

**Comment by jerold** [comment]
Good gawd. This is amazing.

File: golib/install_tools.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
```
Comment (jerold) [inline golib/install_tools.sh:1]: execute perms :)

**Review by jerold — APPROVED** [review]
+10

**Comment by jerold** [comment]
Thanks Ben! This is awesome. :D

**Pull request merged**